packaging/desktop/package_darwin_2.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
+
+set -e # Fail on error
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $dir
+
+client_dir="$dir/../.."
+build_dir="$dir/build"
+tmp_dir="$dir/tmp"
+node_bin="$dir/node_modules/.bin"
+
+app_name=Keybase
+keybase_version=$1
+app_version=$keybase_version
+kbfs_version=$2
+comment=$3
+
+if [ "$keybase_version" = "" ]; then
```

Comment:
:sheep: :sheep: You can do this in one line: https://github.com/keybase/client/blob/master/react-native/uploadApp.sh#L7